vignettes/rmd/Rcpp-FAQ.Rmd

```diff
@@ -1765,3 +1765,12 @@ package, it is generally recommended to have both `Imports: Rcpp` and
 that this symbol is registered when your package is loaded by R, and as a
 side-effect certain other \pkg{Rcpp} function identifiers will also be
 resolved properly.
+
+## On macOS, 'no matching function for call to `R_lsInternal`'
+
+In issue [#1148](https://github.com/RcppCore/Rcpp/issues/1148) an error due
+to overeager includes was reported. Including `Rinternals.h` along with the
+(macOS-only) `mach/boolean.h` lead to linker error as `mach/boolean`
+redefines `TRUE` leading to bad interactions with the `Rboolean` enum type. A
+very simple solution is to be more careful and conservative with `#include`
+files and a) have `#include <mach/boolean.h>` appear first and b) skip
```

Comment:
Did this sentence end prematurely?

Reply:
Uggh. I am having some minor fights with the md mode in emacs reverting. Will need to check.

Reply:
Grrr. Yes. Lacks "$include <Rinternals.h>". My bad.

Reply:
Suggest adding `the 

```
the `#include <Rinternals.h>` as it is included by `Rcpp.h` anyway.
```